packages/web3torrent/src/components/torrent-info/upload-info/UploadInfo.stories.tsx

```diff
@@ -27,7 +27,8 @@ storiesOf('Web3Torrent', module)
             },
             funds: '50',
             allowed: true,
-            seederBalance: '50'
+            seederBalance: '50',
+            channelId: '0x00012319283712'

```

Comment:
In general I like mock data to mock the structure of the real data. In this case it would be a length 42 string.. maybe use `0x0000000000000000000000000000001231927371`

Reply:
okay! fixed